frontend/src/views/Unauthorized.vue

```diff
@@ -0,0 +1,16 @@
+<template>
+  <div>
+    <H1>Access Denied</H1>
```

Comment:
h1 tag should be lower-case.

Reply:
Yes it should, not sure why I capitalized it.